### PR TITLE
Style Select!

### PR DESCRIPTION
**Pull request by PixelRifts**
The basics are finally done!

### CODE REVIEW

**Comment by ambrosiogabe** [comment]
Wooh! Thanks Nikhil! I'm gonna take a closer look at your code tomorrow morning and let you know if there's any changes I think you should make. From my cursory glance though it looks good :)

File: JadeEditor/cpp/MenuBar.cpp [hunk 1]
```diff
@@ -89,6 +116,12 @@ namespace Jade
 					m_SettingsOpen = true;
 				}
 
+				if (JImGui::MenuButton("Styles"))
+				{
+					// Maybe move this inside settings??
```
Comment (PixelRifts) [inline JadeEditor/cpp/MenuBar.cpp:121]: Ignore this....

**Comment by PixelRifts** [comment]
Btw I know you're planning to (maybe already doing) to embed C# as a scripting language.
Maybe I can try to also add Java as one. I know they are similar, but most of the viewers of your channel are java programmers.
(If you dont want it just say no. I have no problems with that). It will only mostly be on my fork of this project

**Comment by PixelRifts** [comment]
Btw I know you're planning to (maybe already doing) to embed C# as a scripting language 
Maybe I can try to also add Java as one. I know they are similar, but most of the viewers of your channel are java programmers.
(If you dont want it just say no. I have no problems with that). It will only mostly be on my fork of this project

**Review by ambrosiogabe — APPROVED** [review]
Everything looks good to me, I may refactor some of it in the future so that there is a strict dependency chain on loading styles through the ImGuiLayer, instead of having a public static method. But it's fine for now :)

**Pull request merged**